### PR TITLE
feat: Add option to change password.

### LINKED ISSUE #1355: Add change password fragment
**Describe the feature you'd like**
Add change password fragment

**Would you like to work on the issue?**
-  Yes

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #1355 

Changes: Added option to change password in settings.

Screenshots for the change:
image
image

### CODE REVIEW

**Comment by angmas1** [comment]
@liveHarshit @nikit19 please review

File: app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt [hunk 1]
```diff
@@ -222,7 +222,7 @@ class SignUpFragment : Fragment() {
                     textInputLayoutPassword.isEndIconVisible = true
                 }
 
-                if (passwordSignUp.text.toString().length >= 6 || passwordSignUp.text.toString().isEmpty()) {
+                if (passwordSignUp.text.toString().length >= 6) {
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt]: Why?

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt]: Unneeded second case.

**Comment by angmas1** [comment]
@iamareebjamal please review

**Comment by angmas1** [comment]
@iamareebjamal please review

**Comment by iamareebjamal** [comment]
Peer review

**Comment by angmas1** [comment]
@liveHarshit reviewed

**Comment by angmas1** [comment]
Please review before PR becomes stale.

**Pull request merged**